src/TwitterFeed.php

```diff
@@ -94,6 +96,7 @@ protected function getFeed($count = 5)
                 "count" => $count,
                 "exclude_replies" => $this->excludeReplies,
                 'include_rts' => $this->includeRts,
+                'tweet_mode' =>  ($this->extended ? 'extended' : '')
```

Comment:
@pcgc Does this option could be useful for TwitterSearchFeed too?

Reply:
Twitter search api doesn't offer this

https://developer.twitter.com/en/docs/tweets/search/api-reference/get-search-tweets

Reply:
this is now supported by search api too